functions/__fnm_run_bin_as.fish

```diff
@@ -48,12 +48,15 @@ function __fnm_run_bin_as -a name
 
     if test "$name" = "node"
         command node $argv
+        set command_status $status
 
     else if test "$name" = "npm"
         command npm $argv
+        set command_status $status
 
     else
         echo "fnm: Error: I don't know what is '$name'." > /dev/stderr
+        set command_status 1
     end
 
     if test ! -z "$rc_ver"
```

Comment:
Not sure if it is safe to move these lines upper. But if so there is no need to introduce a `command_status` variable, exit code will be passed implicitly.